src/View/Requirements_Backend.php

```diff
@@ -1421,6 +1439,39 @@ function () use ($fileList, $minify, $type) {
         return $combinedURL;
     }
 
+    /**
+     * Resolves relative paths in CSS files which are lost when combining them
+     *
+     * @param string $content
+     * @param string $filePath
+     * @return string New content with paths resolved
+     */
+    protected function resolveCSSReferences($content, $filePath)
+    {
+        $doResolving = Config::inst()->get(__CLASS__, 'resolve_relative_css_refs');
+        if (!$doResolving) {
+            return $content;
+        }
+        $fileUrl = Injector::inst()->get(ResourceURLGenerator::class)->urlForResource($filePath);
+        $fileUrlDir = dirname($fileUrl);
+        $content = preg_replace_callback('#([\s\'"\(])((:?[\.]{1,2}/)+)#', function ($a) use ($fileUrlDir) {
+            [ $_, $prefix, $match ] = $a;
```

Comment:
These variables _all_ represent matches, so `$match` isn't a very descriptive name here. `$relativePath` is probably a more appropriate variable name instead of `$match`.